display_list/display_list_unittests.cc

```diff
@@ -414,8 +416,9 @@ std::vector<DisplayListInvocationGroup> allGroups = {
     }
   },
   { "SetPathEffect", {
-      {0, 16, 0, 0, [](DisplayListBuilder& b) {b.setPathEffect(TestPathEffect1);}},
-      {0, 16, 0, 0, [](DisplayListBuilder& b) {b.setPathEffect(TestPathEffect2);}},
+      // 2 * sizeof(SkScalar)
```

Comment:
plus the size of the DlPathEffect object + the overhead of the SetPathEffectOp (probably 8 bytes)